packages/api/cms-api/src/user-permissions/decorators/scoped-entity.decorator.ts

```diff
@@ -1,13 +1,17 @@
-import { CustomDecorator, SetMetadata } from "@nestjs/common";
+import { AnyEntity } from "@mikro-orm/core";
+import { CustomDecorator, SetMetadata, Type } from "@nestjs/common";
 
 import { ContentScope } from "../../user-permissions/interfaces/content-scope.interface";
 
-export interface ScopedEntityMeta {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    fn: (entity: any) => Promise<ContentScope | ContentScope[]>;
+export type GetEntityScope<Entity extends AnyEntity = AnyEntity> = (
```

Comment:
I'm unsure about the naming in this file. Multiple things sound similar, but are different. What do you think?

Reply:
it's understandable to me.

Maybe `GetEntityScope` -> `GetEntityScopeFn` or  `GetEntityScopeInlineFn` or  `GetEntityScopeFunction`?